src/plugins/feeds/feeds.js

```diff
@@ -243,11 +251,76 @@ var componentName = "wb-feeds",
 		}
 	},
 
+	/**
+	 * Process Feed/JSON Entries for CORS Enabled
+	 * @method corsEntry
+	 */
+	corsEntry = function( xmlDoc, limit ) {
+		var arr_entry = [];
```

Comment:
Do one "var" declaration instead of multiple, this is to follow the same coding style as we did elsewhere in WET. This apply to other "var" statement you added in your PR.